File: fm-domain/src/main/java/io/zeitwert/fm/app/adapter/api/rest/dto/TenantInfoResponse.java
package io.zeitwert.fm.app.adapter.api.rest.dto;

import lombok.Builder;
import lombok.Data;

import java.util.List;

import io.zeitwert.dddrive.ddd.api.rest.dto.EnumeratedDto;

@Data
@Builder
public class TenantInfoResponse {

	private Integer id;
	private EnumeratedDto tenantType;
	private List<EnumeratedDto> accounts;

}

File: fm-domain/src/main/java/io/zeitwert/fm/app/adapter/api/rest/dto/TenantInfoResponse.kt
package io.zeitwert.fm.app.adapter.api.rest.dto

import io.zeitwert.dddrive.ddd.adapter.api.jsonapi.dto.EnumeratedDto

data class TenantInfoResponse(
	val id: Int,
	val tenantType: EnumeratedDto,
	val accounts: List<EnumeratedDto>,
)
